src/molecules/Carousel.tsx

```diff
@@ -0,0 +1,152 @@
+import type { FunctionComponent, ReactNode } from 'react';
+import { useEffect, useState } from 'react';
+
+import type { FlexProps } from '../atoms';
+import { Box, Flex, Heading, Image, SubHeading } from '../atoms';
+
+export interface CarouselElement {
+  title: string;
+  text: ReactNode;
+  icon: string;
+}
+
+export interface CarouselProps {
+  title: string;
+  inverted?: boolean;
+  elements: CarouselElement[];
+
+  /**
+   * The interval between switching elements. Set to 0 to disable.
+   */
+  interval?: number;
+}
+
+interface CarouselHeadingProps {
+  title: string;
+  index: number;
+  isActive: boolean;
+  onClick?(index: number): void;
+}
+
+const CarouselHeading: FunctionComponent<CarouselHeadingProps> = ({
+  title,
+  index,
+  isActive,
+  onClick
+}) => {
+  const handleClick = () => onClick?.(index);
+
+  return (
+    <SubHeading
+      key={title}
+      onClick={handleClick}
+      flex="0 0 auto"
+      fontSize={['3', null, '4']}
+      color={isActive ? 'carousel.active' : 'text.discrete'}
+      my="3"
+      mr="4"
+      lineHeight="150%"
+      sx={{
+        '&:first-child': {
+          ml: ['4', null, 0]
+        },
+        '&:last-child': {
+          mr: ['4', null, 0]
+        },
+        whiteSpace: 'nowrap',
+        cursor: 'pointer',
+        borderBottom: '2px solid',
+        borderBottomColor: isActive ? 'carousel.active' : 'transparent'
+      }}>
+      {title}
+    </SubHeading>
+  );
+};
+
+export const Carousel: FunctionComponent<CarouselProps & FlexProps> = ({
+  title,
+  elements,
+  inverted = false,
+  interval = 7500,
+  ...props
+}) => {
+  const [index, setIndex] = useState(0);
+
+  useEffect(() => {
+    if (interval > 0) {
```

Comment:
Can we also disable interval on `undefined`?

Reply:
```
> undefined > 0
< false
```

Reply:
Right 👍